docs/pages/data-fetching/+Page.mdx

```diff
@@ -10,31 +10,37 @@ The usual recommendation is:
 
 You can fetch the initial data of a page by using <Link href="/data">Vike's `+data` hook</Link>, then access it by using the <Link href="/useData">component hook `useData()`</Link>.
 
-```js
-// /pages/movies/@id/+data.js
+```ts
+// /pages/movies/@id/+data.ts
 // Environment: server
 
 export { data }
+export type Data = Awaited<ReturnType<typeof data>>
 
 import fetch from 'node-fetch'
+import type { PageContextServer } from 'vike/types'
 
-async function data(pageContext) {
+type MovieDetails = {
+  // ...
+}
+
+async function data(pageContext: PageContextServer) {
   const { id } = pageContext.routeParams
   const response = await fetch(`https://star-wars.brillout.com/api/films/${id}.json`)
 
-  let movie = await response.json()
+  let movie: MovieDetails = await response.json()
```

Comment:
```suggestion
  let movie = await response.json()
```

Hm. I think it's okay if we write TypeScript that isn't completely valid (or has some implicit `any` values). The overall goal here is to teach users TypeScript usage in the context of using Vike. I'd argue we don't have to (nor should we) teach TypeScript in general. Let's assume they know TypeScript well (if they don't they will be able to toggle to JSX).